src/components/exchange/NetworkSwitcherv2.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useEffect, useRef } from 'react';
+import { ScrollView } from 'react-native-gesture-handler';
```

Comment:
i missed this during my first review, any special reason to use `react-native-gesture-handler` instead of `react -native`

Reply:
fixed